app/client/components/pages/dashboardPage.jsx

```diff
@@ -33,8 +33,7 @@ const mapStateToProps = (state) => {
 const mapDispatchToProps = (dispatch) => {
   return {
     logout:       () => dispatch(logout()),
-    loadCreditTransferList: (query, path) => dispatch(loadCreditTransferList({query, path})),
-    activateAccount: (activation_token) => dispatch(activateAccount(activation_token))
+    loadCreditTransferList: (query, path) => dispatch(loadCreditTransferList({query, path}))
```

Comment:
this api request payload can get standardized too once more typescript conversions are performed